New version Prestashop 4.4.4 (#1043)
mercadopago/devsite-docs: guides/resources/changelog/current-year.en.md

## January 2021

> ### January 28nd

Reviewer: ```suggestion
### January 28th
```

> CHANGELOG
>
> New PrestaShop plugin version
>
> NEW_VERSION: NEW VERSION
>
> PRODUCT: PRESTASHOP

> We have released the new version 4.4.4 for PrestaShop.

Reviewer: ```suggestion

We have released the new version 4.4.4 for PrestaShop.
```

[Go to PrestaShop documentation](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/plugins/prestashop/introduction)

> GIT
>
> GitHub
>

> > [Check GitHub](https://github.com/mercadopago/cart-prestashop-7/releases/tag/v4.4.4) fo

Reviewer: ```suggestion
> [Check GitHub](https://github.com/mercadopago/cart-prestashop-7/releases/tag/v4.4.4) for further details regarding this release.
```
